assets/stylesheets/index.css

```diff
@@ -0,0 +1,248 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #eee;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  width: 100%;
+  height: auto;
+  margin-top: 0.6em;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.signin-btn {
+  width: 7.8em;
+  height: 35px;
+  border-radius: 5px;
+  padding-right: 8px;
+  margin-left: 10px;
+}
+
+.signup img {
+  display: inline-block;
+  width: 16px;
+  height: 12px;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+}
+
+.mini-header {
+  margin-top: 20px;
+}
+
+.header img {
+  display: inline-block;
+  width: 16px;
+  height: 12px;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+}
+
+.header div {
+  line-height: 20px;
+  text-align: right;
+  font-size: 12px;
+}
+
+.header a {
+  color: #0077c5;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.logo img {
+  width: 140px;
+  padding: 10px;
+}
+
+.sub-logos {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.sub-logos img {
+  width: 24px;
+  height: 24px;
+  margin-right: 2px;
+}
+
+.sub-logos span {
+  margin: 4px;
+  color: #999;
+}
+
+.main-wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+form {
+  width: 30%;
+  height: auto;
+  border: 1px solid #aaa;
+  background: white;
+  font-size: large;
+  font-weight: 200;
+  margin-top: 1em;
+  border-radius: 3px;
+  text-align: center;
+  color: #444;
+}
+
+.info {
+  margin: 0 1.5em;
+}
+
+.learn-more {
+  text-decoration: none;
+  color: #0077c5;
+}
+
+ul {
+  list-style-type: none;
+}
+
+ul li {
+  text-align: center;
+  height: 130px;
+}
+
+label {
+  display: block;
+  text-align: left;
+  margin-left: 20px;
+  font-size: 14px;
+  line-height: 30px;
+}
+
+input {
+  height: 3em;
+  border-radius: 5px;
+  border: 1px solid #c1c5c8;
+}
+
+.email {
+  background: white;
+}
+
+.number {
+  background: white;
+}
+
+.password {
+  background: white;
+}
+
```

Comment:
[Required] Remove all the repeated styling. you can do something like this :point_down: 
`.email, .number, .password {
  background: white;
}`